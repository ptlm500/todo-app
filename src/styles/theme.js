export default {
  background: '#FFFFFF',
  primaryDark: '#303F9F',
  primaryLight: '#C5CAE9',
  primary: '#3F51B5',
  disabled: '#BDBDBD',
  divider: '#BDBDBD',
  text: '#212121',
  icon: '#212121',
  invertedText: '#FFFFFF',
  invertedIcon: '#FFFFFF',
  secondaryText: '#757575',
  secondaryIcon: '#757575',
  accent: '#448AFF',
  danger: '#D32F2F',
  dangerLight: '#F44336',
  green: '#388E3C',
  font: '\'Segoe UI\' \'HelveticaNeue-Light\' sans-serif',
  defaultRadius: '0.5rem',
  shortTransition: '0.1s',
  longTransition: '0.3s'
};
