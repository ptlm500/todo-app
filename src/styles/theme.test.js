import theme from './theme';

describe('theme', () => {
  it('matches the snapshot', () => {
    expect(theme).toMatchSnapshot();
  });
});
