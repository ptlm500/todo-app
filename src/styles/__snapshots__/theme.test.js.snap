// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme matches the snapshot 1`] = `
Object {
  "accent": "#448AFF",
  "background": "#FFFFFF",
  "bodyFontSize": "
    @media (max-width: 480px) {
      font-size: 12px;
    }
    
      @media (min-width: 640px) {
        font-size: 16px;
      }
    
    font-size: 2.5vw
  ",
  "danger": "#D32F2F",
  "dangerLight": "#F44336",
  "defaultRadius": "0.5rem",
  "disabled": "#BDBDBD",
  "divider": "#BDBDBD",
  "font": "'Segoe UI' 'HelveticaNeue-Light' sans-serif",
  "green": "#388E3C",
  "headerFontSize": "
    @media (max-width: 240px) {
      font-size: 12px;
    }
    
      @media (min-width: 640px) {
        font-size: 32px;
      }
    
    font-size: 5vw
  ",
  "icon": "#212121",
  "invertedIcon": "#FFFFFF",
  "invertedText": "#FFFFFF",
  "longTransition": "0.3s",
  "primary": "#3F51B5",
  "primaryDark": "#303F9F",
  "primaryLight": "#C5CAE9",
  "red": "#D32F2F",
  "secondaryIcon": "#757575",
  "secondaryText": "#757575",
  "shortTransition": "0.1s",
  "text": "#212121",
}
`;
